ray-operator/test/e2eautoscaler/support.go

```diff
@@ -0,0 +1,132 @@
+package e2eautoscaler
+
+import (
+	"embed"
+
+	"github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	corev1ac "k8s.io/client-go/applyconfigurations/core/v1"
+
+	rayv1ac "github.com/ray-project/kuberay/ray-operator/pkg/client/applyconfiguration/ray/v1"
+	. "github.com/ray-project/kuberay/ray-operator/test/support"
+)
+
+//go:embed *.py
+var _files embed.FS
```

Comment:
Instead of refactoring and sharing the old code, I copy the necessary code from the `test/e2e/support.go` to the new `test/e2eautoscaler/support.go` now for easy review with minimum code changes.

Reply:
Just for my understanding, could you please explain why we need to refactor the code from `test/e2e/support.go`?

Reply:
I suggest that we should not refactor the code from `test/e2e/support.go` but just copy the necessary code instead to keep the changes of this PR smaller and simpler.

Reply:
@rueian So there will be another PR to do the refactoring?

Reply:
Perhaps I'm missing something, but I'm curious why we can't simply share them

Reply:
Hi @Yicheng-Lu-llll, here is a preview of making them shareable. The changes are quite large, touching 11 files.

Reply:
Oh, I see! Personally, I prefer the #2207 approach since only one copy needs to be maintained, but I'm also okay with this approach. Let's have @kevin85421 decide.